app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -129,31 +165,47 @@ export default {
         sku: null,
         price: '',
         currency: 'CLP',
-        quantity: '',
-        measure: '',
+        ingredient_measures_attributes: [],
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Identifier 'ingredient_measures_attributes' is not in camel case  camelcase